src/microbe_stage/MicrobeStage.cs

```diff
@@ -451,20 +460,28 @@ public override void _Process(float delta)
             TutorialState.SendEvent(TutorialEventType.MicrobePlayerTotalCollected,
                 new CompoundEventArgs(Player.TotalAbsorbedCompounds), this);
 
-            elapsedSinceCompoundPositionCheck += delta;
+            elapsedSinceEntityPositionCheck += delta;
 
-            if (elapsedSinceCompoundPositionCheck > Constants.TUTORIAL_COMPOUND_POSITION_UPDATE_INTERVAL)
+            if (elapsedSinceEntityPositionCheck > Constants.TUTORIAL_ENTITY_POSITION_UPDATE_INTERVAL)
             {
-                elapsedSinceCompoundPositionCheck = 0;
+                elapsedSinceEntityPositionCheck = 0;
 
                 if (TutorialState.WantsNearbyCompoundInfo())
                 {
                     TutorialState.SendEvent(TutorialEventType.MicrobeCompoundsNearPlayer,
-                        new CompoundPositionEventArgs(Clouds.FindCompoundNearPoint(Player.GlobalTransform.origin,
+                        new EntityPositionEventArgs(Clouds.FindCompoundNearPoint(Player.GlobalTransform.origin,
                             glucose)),
                         this);
                 }
 
+                if (TutorialState.WantsNearbyEngulfableInfo())
+                {
+                    var engulfables = entities.OfType<IEngulfable>().ToList();
```

Comment:
The entities variable should only be initialized here to save on performance as querying the Godot node tree is not free.

Reply:
`entities` is also used for debug overlay in this method's bottom-most.

Reply:
I see, I think in this case either code duplication or extracting that as a method would be better, as these both are special use cases which should not be made cost performance each frame, that's entirely why the tutorial positions aren't updated each frame to save on performance.